debian/control

```diff
@@ -0,0 +1,15 @@
+Source: imx-firmware
+Section: contrib/electronics
+Priority: optional
+Build-Depends: debhelper-compat (= 12)
+Standards-Version: 3.9.8
+Maintainer: KUNBUS GmbH <support@kunbus.com>
+Homepage: https://revolution.kunbus.com/
+
+Package: firmware-nxp
+Architecture: all
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: Revolution Pi - firmware-nxp
+ Firmware file to MAYA-W1 module.
+ Revolution Pi is a set of IoT products based on the Raspberry Pi and
+ geared towards industrial usage.
```

Comment:
With a short description there was a warning.
I have updated this with a description of the driver, taken from history... 
I thought it would also be correct to add their firmware version.